app/api/v1/models/election.py

```diff
@@ -46,6 +51,38 @@ class CiphertextElectionContext(Base):
     crypto_extended_base_hash: str
     """The `extended base hash code (𝑄')` in [ElectionGuard Spec](https://github.com/microsoft/electionguard/wiki)"""
 
+    @staticmethod
+    def stringToElementModP(value: str) -> ElementModP:
+        elementModP = hex_to_p(value)
+        if elementModP is None:
+            raise HTTPException(
+                status_code=status.HTTP_400_BAD_REQUEST, detail="invalid key"
+            )
+        return elementModP
+
+    @staticmethod
```

Comment:
These methods in the way they are written should be in their own file and imported here.